src/org/aavso/tools/vstar/ui/dialog/plugin/manager/PluginManager.java

```diff
@@ -754,7 +759,12 @@ public void deletePlugin(String description) {
 					// vagaries of file systems or the possibility of
 					// concurrent deletion.
 					if (libJarPath.exists()) {
-						// if (libJarPath.delete()) {
+						// Max: This fragment was commented out by David(?)
+						// Even being commented out, it contained an error in the if-condition:
+						// logical '!' was missing!
+						// I have not uncommented it but fixed the error.
+						// So, actually, we never delete the libraries. 
+						// if (!libJarPath.delete()) {
```

Comment:
That's right. See also my comment above. Arguably we could allow this deletion to occur now that we don't use WebStart, if that is indeed the reason. Speculation at the moment.